addons/im_livechat/models/discuss_channel.py

```diff
@@ -149,8 +180,9 @@ def _get_channel_history(self):
         """
         Converting message body back to plaintext for correct data formatting in HTML field.
         """
+        # sudo - res.partner: public user can access internal user messages
```

Comment:
This comment is misleading. Accessing internal user messages from a public user is concerning and likely unintended. However, in this case, you’re only accessing the operator’s name, which makes sense as it’s reasonable for visitors to see the live chat operator’s name.